### PR TITLE
[LMS-290] Getting started landing page

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: vm-docs/index.html [hunk 1]
```diff
@@ -57,55 +53,426 @@
...
+            evaluating how engaged students are in their learning - be it remote, in-person, or hybrid. This Starter
+            Kit, leveraging the <a href="https://techdocs.ed-fi.org/x/_rf_B">LMS Toolkit</a>, provides a solution
+            environment in which data from an LMS
+            is collected and combined with student demographic and assessment data from other key systems to help paint
+            a more accurate picture of student engagement.</p>
+        <h2>Contents</h2>
+        <ol>
+            <li><a href="#beginning">Beginning at the End</a></li>
+            <li><a href="#understanding">Understanding the Components of the Starter Kit</a></li>
+            <li><a href="#using">Using the Starter Kit Components</a>
```
Comment (lalrg) [inline vm-docs/index.html:140]: This #using link is not scrolling down
The #platform is also not scrolling down

File: vm-docs/index.html [hunk 2]
```diff
@@ -57,55 +53,426 @@
...
+            <li>ODS Admin App, providing administrative management of the API and ODS, </li>
+            <li>Swagger UI, a web application that documents the API and allows you to run sample calls against it,
+            </li>
+            <li>Console Bulk Loader, a console (command line) application for bulk uploading xml
+                files into the Ed-Fi API.</li>
+        </ul>
+        <div class="centered"><img
+                alt="Diagram showing the relationships between the ODS/API, LMS Toolkit, and Power BI dashboards."
+                class="framed" src="eolsk-diagram.png"></div>
+        <h2>Using the Starter Kit Components</h2>
```
Comment (stephenfuqua) [inline vm-docs/index.html:198]:
```suggestion
        <h2 id="using">Using the Starter Kit Components</h2>
```

File: vm-docs/index.html [hunk 3]
```diff
@@ -57,55 +53,426 @@
...
+        <h3>Ed-Fi Technology Platform</h3>
```
Comment (stephenfuqua) [inline vm-docs/index.html:199]:
```suggestion
        <h3 id="platform">Ed-Fi Technology Platform</h3>
```

File: vm-docs/index.html [hunk 4]
```diff
@@ -57,55 +53,426 @@
...
+        }
+
+        .centered {
+            text-align: center;
+        }
     </style>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/vs2015.min.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/powershell.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/sql.min.js"></script>
```
Comment (stephenfuqua) [inline vm-docs/index.html:118]:
```suggestion
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/vs2015.min.css">
    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/powershell.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/sql.min.js"></script>
```

**Review by lalrg — APPROVED** [review]
Looks good!

**Pull request merged**